Framework/Core/include/Framework/InputSpec.h

```diff
@@ -79,5 +82,7 @@ struct InputSpec {
   bool operator==(InputSpec const& that) const;
 };
 
+void updateInputList(std::vector<InputSpec>& list, InputSpec&& input);
```

Comment:
Also this one should be moved to the utilities.

Reply:
In which header should I put it then?

Reply:
See above, either DataDescriptorQueryBuilder or DataSpecUtils.